### DISCUSSION TITLE
What is the limit option suggested by the docs?

### DISCUSSION THREAD

**Discussion by captn3m0** [mitmproxy/mitmproxy#6681] [Q&A]
The documentation at https://docs.mitmproxy.org/stable/howto-ignoredomains/ says:

> Note that mitmproxy’s “Limit” option is often the better alternative here, as it is not affected by the limitations listed below.

Is this refering to the [` view_filter`](https://docs.mitmproxy.org/stable/concepts-options/#view_filter) option? I couldn't find a relevant limit option in the documentation.

**Comment by mhils** [comment]
See https://github.com/mitmproxy/mitmproxy/issues/6682!
